api/data/it/PD.tsx

```diff
@@ -420,7 +442,7 @@ export const StripeLink = () => {
                         </Typography>
                     </Grid>
                     <Grid item xs={12} md={6}>
-                        <Button variant="contained" color="primary" endIcon={<ArrowForwardIcon />} href="https://developer.pagopa.it/send/overview">
+                        <Button variant="contained" color="primary" endIcon={<ArrowForwardIcon />} href="https://docs.pagopa.it/send-kit-di-comunicazione-per-gli-enti-aderenti/">
```

Comment:
@salvatorediocaro the correct link for this item is "https://developer.pagopa.it/send/overview"